src/applications/mhv-medical-records/tests/e2e/accelerated/pages/Allergies.js

```diff
@@ -20,6 +21,12 @@ class Allergies {
       }
       req.reply(allergiesData);
     }).as('allergies-list');
+    // Note that we don't need specific event names in the response
```

Comment:
Note: I add the intercept for UUM to get rid of any errors, but we don't test it in the E2E tests.